src/shared/form/YesNoDetailsField.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Control } from 'react-redux-form';
+
+const YesNoDetailsField = ({ name, id, label, model, validators, messages, description, controlProps, mapProps }) => (
+  <div className="field">
+          <Control.checkbox
+                 name={name}
+                 id={id}
+                 value="Yes"
+                 model={model}
+          />
+          <label htmlFor={id}>
+              {label}
+          </label>
+          If yes, provide details...
+          <Control.textarea
+              name={`${name}_details`}
+              id={`${id}-details`}
+              model={`${model}_details`}
+          >
+          </Control.textarea>
+  </div>
```

Comment:
If I click yes on the checkbox, should user be forced to enter text in textarea?